### DISCUSSION TITLE
Clean/Truncate all logs?

### DISCUSSION THREAD

**Discussion by sungkhum** [EasyEngine/easyengine#1703] [Support Requests]
I'm having some major issues with log size on my easy engine v4 install. I only have one site. Is there a way to clean these logs?

`12025751831 Jun  2 21:26 /var/lib/docker/volumes/global-nginx-proxy_nginx_proxy_logs/_data/access.log`

`9695418199 Jun  2 21:26 /var/lib/docker/volumes/sitename_log_nginx/_data/access.log`

`100663296 Jun  2 21:26 /var/lib/docker/volumes/global-db_db_data/_data/ib_logfile0`

`302028852 Jun  2 21:26 /var/lib/docker/volumes/sitename_log_php/_data/access.log`

**Comment by dhsathiya** [comment]
Try this out. We will implement this soon on EasyEngine :crossed_fingers: 
https://github.com/EasyEngine/feature-requests/issues/73#issuecomment-667002783
